lib/maze/client/appium/bs_client.rb

```diff
@@ -31,6 +30,11 @@ def device_capabilities
           unless device_caps['platformName'] == 'android' && device_caps['platformVersion'].to_i <= 6
             capabilities['bstack:options']['disableAnimations'] = 'true'
           end
+          if Maze.config.start_tunnel
+            capabilities['bstack:options']['local'] = 'true'
+            capabilities['bstack:options']['localIdentifier'] = @session_uuid
+          end
```

Comment:
Is there any possibility of breaking things because this is now after the deep merges?

I'm guessing no, since it wouldn't work without the local tunnel and needs this setup either way

Reply:
As long as `deep_merge!` didn't do something annoying like remove the empty hash for `bstack:options` I think it should be fine.  Maze Runner's CI does cover use of the BrowserStack tunnel though (with both W3C and JWP protocols), so we would notice a breakage now and I don't think it can shift beneath us.